app/experimenter/experiments/bugzilla.py

```diff
@@ -162,3 +159,14 @@ def create_experiment_bug(experiment):
 def get_bugzilla_id(bug_url):
     query = urlparse(bug_url).query
     return int(parse_qs(query)["id"][0])
+
+
+def set_bugzilla_id_value(bug_url):
+    if "show_bug.cgi?id=" in bug_url:
```

Comment:
Also if you parse the qs and use `.get` I don't think you need to do this check either.